### DISCUSSION TITLE
Published content does not display - returns "No content found" or "Too many requests"

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#607] [2. Published content] [answered]
For one of our published docs, https://docs.tonic.ai/textual, users are not seeing the content - it's returning the message "No content found" or "Too many requests".

I had the guide open earlier, but ran into the same error once. Then it worked.

But other users are still seeing the error.

**Comment by aikoii** [comment]
@JaniceManwiller thanks for flagging! We are working on a fix and will keep you updated 🙏

**Reply by aikoii** [reply, marked as answer]
We released a fix for this issue 🤞. Please let me know if you're still experiencing any problems.
​
